internal/apiserver/perms.go

```diff
@@ -117,7 +116,6 @@ func initializePerms() {
 		CLONE_PERM:        "yes",
 		DF_CLUSTER_PERM:   "yes",
 		LABEL_PERM:        "yes",
-		LOAD_PERM:         "yes",
```

Comment:
If a role has been granted this permission during an earlier version, will the role continue to work (for other granted functionality) after an upgrade?

Reply:
Well it wouldn't be considered at all.

Reply:
Maybe something our upgrade code should take care of, i.e. removing references to this now-defunct permission?